Libraries/LibC/stdio.cpp

```diff
@@ -440,6 +454,19 @@ void FILE::Buffer::drop()
     m_ungotten = false;
 }
 
+size_t FILE::Buffer::buffered_size() const
+{
+    // Note: does not include the ungetc() buffer.
+
+    if (m_empty)
+        return 0;
+
+    if (m_begin < m_end)
+        return m_end - m_begin;
+    else
```

Comment:
No need for `else` after `return`.

Reply:
Yes, but it also doesn't hurt. This isn't an early return in some simple case (like say `m_empty` check above); the two case are equally important, so it makes sense to treat them both equally.

Reply:
Meh, if you insist. It's just unnecessary cognitive burden to me :)

Reply:
Thanks for the fix btw, looks great!